api/src/main/java/org/openmrs/api/db/ConceptDAO.java

```diff
@@ -398,24 +417,12 @@ public Integer getCountOfConcepts(String phrase, List<Locale> locales, boolean i
 	public ConceptSource getConceptSourceByHL7Code(String hl7Code);
 
 	/**
-	 * Gets the value of conceptDatatype currently saved in the database for the given concept,
-	 * bypassing any caches. This is used prior to saving an concept so that we can change the obs
-	 * if need be
-	 * 
-	 * @param concept for which the conceptDatatype should be fetched
-	 * @return the conceptDatatype currently in the database for this concept
-	 * @should get saved conceptDatatype from database
```

Comment:
Any reason for removing this should annotation? And many others?

Reply:
Hi @dkayiwa I have removed those @should given that it became an @see. I have rechecked the purpose of the @should and that it would be necessary for unit testing, if I'm not mistaken, please correct me. :D

However, given that it became @see, the @should is not necessary anymore, and of which could be tested in another class :)

Reply:
What do you mean by @should became a @see?

Reply:
Hi @dkayiwa! I have replaced the java documentation (which is a duplicate of that in ConceptService.java) into a single @see javadoc, following the documentation of the other functions. :)